### PR TITLE
Ignore the src/pgbackrest that gets built when doing make...

### PR DESCRIPTION
**Pull request by sfrost**
Would be nice to have this so that a 'make' doesn't make it look like there's something to commit. :)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 **/*~
 *~
 *.swp
+src/pgbackrest
```
Comment (dwsteele) [inline .gitignore]: Wouldn't we want to ignore `.o` files, etc., as well?  The tests don't actually build in this directory which is why we haven't done this yet.

Also, I'd prefer to have these exclusions in `src/.gitignore`.

Comment (sfrost) [reply .gitignore]: Oh, perhaps...  My global gitignore includes:

tags
.*.swp
cscope.in.out
cscope.out
cscope.po.out
*.o

I didn't want to put this into my global one though.  I'm fine with adding *.o to the pgbackrest one.

I'm fine with putting the entry wherever works. :)

Comment (dwsteele) [reply .gitignore]: Yeah, add `.o` and move it to `src/.gitignore`.  The idea is if we get rid of dirs (not this one I hope) then all the exclusions go with it.

Comment (sfrost) [reply .gitignore]: ok, updated the branch with that change and did a force-push to it and seems to have correctly updated the PR. :)

**Comment by dwsteele** [comment]
Committed at ed5d7a53de7cd.

**Pull request closed without merging**